packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+$data = [
+    [1, 'One', 'First item'],
+    [2, 'Two', 'Second item'],
+    [3, 'Three', 'Third item'],
+    [4, 'Four', 'Fourth item'],
+    [5, 'Five', 'Fifth item'],
+    [6, 'Six', 'Sixth item'],
+    [7, 'Seven', 'Seventh item'],
+    [8, 'Eight', 'Eight item'],
+    [9, 'Nine', 'Ninth item'],
+    [10, 'Ten', 'Tenth item'],
+    [11, 'Eleven', 'Eleventh item'],
+    [12, 'Twelve', 'Twelfth item'],
+    [13, 'Thirteen', 'Thirteenth item'],
+    [14, 'Fourteen', 'Fourteenth item'],
+    [15, 'Fifteen', 'Fifteenth item'],
+    [16, 'Sixteen', 'Sixteenth item'],
+];
+
+$dataFields = ['id', 'name', 'description'];
+
+$totalCount = count($data);
+
+$start = array_key_exists('start', $_GET) ? $_GET['start'] : 0;
+$limit = array_key_exists('limit', $_GET) ? $_GET['limit'] : 5;
+if (array_key_exists('filter', $_GET)) {
+    // [{"operator":"like","value":"Value","property":"name"}]
+    // [{"operator":"like","property":"id","value":"1"},{"operator":"like","property":"name","value":"F"},{"operator":"like","property":"description","value":"item"}]
```

Comment:
![Seal of approvement](http://1.bp.blogspot.com/-j4U5S6MqPoI/UWBXUjM-vXI/AAAAAAAAYRw/leLBNfZdzBI/s640/Seal-of-approval.jpg)